src/models.py

```diff
@@ -0,0 +1,43 @@
+from init import db
+
+
+class User(db.Model):
```

Comment:
Reccomendation: use alembic or some other system to keep database revisions. They'll make it easier to replicate iterative changes to the db schema in the future